protocol-designer/src/ui/labware/selectors.ts

```diff
@@ -35,30 +39,63 @@ export const _sortLabwareDropdownOptions = (options: Options): Options =>
     return a.name.localeCompare(b.name)
   })
 
-/** Returns options for labware dropdowns.
+const getNickNames = (
```

Comment:
return type is string so getNickName would be a better name.

```suggestion
const getNickName = (
```

Reply:
oh ya good catch!